Python/Get_BBC_News/getNews.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+def NewsFromBBC():
+     # BBC news api 
+    main_url = " https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=4dbc17e007ab436fb66416009dfb59a8"
```

Comment:
You should never be keeping your API Key on your Source Code. Create a `.env` file and make use of `Dotenv` module to pass secret information to your code. Make sure to add a `.env.example` to help other developers know that they can keep their secret keys into that.